h/templates/deform/checkbox_choice.jinja2

```diff
@@ -7,9 +7,11 @@
            class="form-checkbox__label">
       <input type="checkbox"
              name="checkbox"
+             data-ref="formInput"
              value="{{ value }}"
              id="{{ field.oid }}-{{ loop.index0 }}"
-             class="form-checkbox__input {% if field.widget.css_class -%}
+             class="form-checkbox__input
+             {% if field.widget.css_class -%}
```

Comment:
Hmm. I really would like us to be a bit more careful with whitespace controls in templates ([as mentioned previously](https://github.com/hypothesis/h/pull/3827#r77492424)).

It's really not too hard to arrange for a template to satisfy both a) be readable and b) emit minimal additional whitespace when rendered. Here, I'd suggest the following:

```jinja2
             class="form-checkbox__input
             {%- if field.widget.css_class %} {{ field.widget.css_class }}{% endif -%}"
```